__tests__/app.test.js

```diff
@@ -240,7 +240,26 @@ describe("GET /api/articles", () => {
         expect(articles).toBeSorted("created_at", { descending: true });
       });
   });
-  describe("error handling", () => {
+  test("200: Responds with an array of article objects filtered by topic 'cats' when topic 'cats' is specified in query", () => {
```

Comment:
Could you please make this more generic? The test is right, but the description would be better as "filtered by the topic passed" - to prove that we don't just care about cats (well, some of us do, but that's not the point)